nautobot_golden_config/filters.py

```diff
@@ -13,93 +13,9 @@
 from nautobot_golden_config import models
 
 
-class GoldenConfigFilter(CreatedUpdatedFilterSet):
-    """Filter capabilities for GoldenConfig instances."""
+class GenericPlatformFilterSet(CreatedUpdatedFilterSet):
```

Comment:
As I understand it, `CreatedUpdatedFilterSet` is generally used as a mixin, not as a standalone base class. Consider also inheriting from `nautobot.utilities.filters.BaseFilterSet` as well.

Also, a filterset for any model that supports custom fields (it appears most/all of yours do) should also inherit from `nautobot.utiltiies.filters.CustomFieldModelFilterSet`.